diff --git a/README.md b/README.md
index d90ac0e..b19ff32 100644
--- a/README.md
+++ b/README.md
@@ -28,6 +28,7 @@ Apps with a known pairing file location:
 - [Auto Capture](https://apps.apple.com/us/app/dev-auto-capture/id6755616902)
 - [StosDebug](https://github.com/stossy11/StosDebug)
 - [StikStore](https://stikstore.app/)
+- [MochiLog](https://github.com/MochiLog/MochiLog)
 
 ## Prerequisites
 
diff --git a/src/known_apps.rs b/src/known_apps.rs
index 1761bb8..807a712 100644
--- a/src/known_apps.rs
+++ b/src/known_apps.rs
@@ -12,6 +12,7 @@ const LOCKDOWN: &[KnownApp] = &[
     ("Feather", PLIST),
     ("Ksign", PLIST),
     ("LiveContainer", LIVE_CONTAINER),
+    ("MochiLog", PLIST),
     ("Protokolle", PLIST),
     ("SideStore", ALT),
     ("SparseBox", PLIST),
@@ -26,6 +27,7 @@ const REMOTE: &[KnownApp] = &[
     ("Feather", PLIST),
     ("Ksign", PLIST),
     ("LiveContainer", LIVE_CONTAINER),
+    ("MochiLog", PLIST),
     ("Protokolle", PLIST),
     ("Reynard", PLIST),
     ("SideStore", ALT),
